cognite/neat/utils/cdf_loaders/data_classes.py

```diff
@@ -0,0 +1,118 @@
+from abc import ABC
+from dataclasses import dataclass
+from typing import Any, cast
+
+from cognite.client import CogniteClient
+from cognite.client.data_classes._base import (
+    CogniteResourceList,
+    WriteableCogniteResource,
+    WriteableCogniteResourceList,
+)
+
+# The Table, TableWrite data classes in the Cognite-SDK lacks the database attribute.
```

Comment:
Would this eventually find its way to `cognite-sdk` ?

Reply:
Yes, hopefully, but the problem is that would be a breaking change (to the poor RAW implementation), so it would be breaking the SDK.